schema/profile.ts

```diff
@@ -18,6 +18,7 @@ export const saveSettingsSchema = z.object({
   websiteUrl: z.string().url().optional().or(z.literal("")),
   emailNotifications: z.boolean().optional(),
   newsletter: z.boolean().optional(),
+  email: z.string().email().optional().or(z.literal("")),
```

Comment:
Shouldn't the user always have an email or am I missing something? 🤔

Reply:
Let me try without having this once. Should work without this. Also I believe this email not required for updating the form associated to the settings form.

Reply:
Ok sorted out ✅